### ISSUE TITLE
ackermann_steering_controller: electronic 'differential'

### ISSUE THREAD

**Issue by Timple** [ros-controls/ros_controllers#576]
### Background:
We have a vehicle with ackermann steering. The rear-wheels are independent driven with a single motor per rearwheel. 
This becomes more common now that electric vehicles are a thing.

### Proposal:
Would an extension of the ackermann drive be accepted where independent rear-wheels are an option? 

### Considerations:
Backwards compatibility with the existing parameter set would of course be taken into account.

**Comment by bmagyar** [comment]
Absolutely! Just make sure that the default behaviour is maintained as-is to avoid breaking things for other people. A feature flag to enable this would work nicely.
